### PR TITLE
feat(config): import Zooz ZEN14 from zwa templated

### PR DESCRIPTION
**Pull request by alvinchen1**
Don't have a ZEN14, but based the two outlets on the ZEN25 which is labeled Left and Right Outlet. Left ZEN14 as Outlet 1 and Outlet 2

### CODE REVIEW

**Comment by alvinchen1** [comment]
@zwave-js-bot fix lint

**Comment by alvinchen1** [comment]
@AlCalzone , followed your instructions and got it.  You were right, I had only forked MASTER before, so definitely a newbie problem.  Thanks for taking the time to explain it to me.

**Comment by alvinchen1** [comment]
Howdy, not really sure why

lint (14.x) Expected — Waiting for status to be reported
Required
lint-zwave (14.x) Expected — Waiting for status to be reported
Required
unit-tests (12.x, ubuntu-latest) Expected — Waiting for status to be reported

Did not complete.  Should I try and close this PR and try again?

**Comment by AlCalzone** [comment]
Let me try just reopening this.

**Comment by AlCalzone** [comment]
Nevermind - misconfiguration on my part.

File: packages/config/config/devices/0x027a/zen14.json [hunk 1]
```diff
@@ -57,118 +57,38 @@
...
-				{
-					"label": "Both outlets on the plug automatically turn on once power is restored (it ignores the status prior to the power outage)",
-					"value": 1
-				},
-				{
-					"label": "Both outlets on the plug remember and restore the status prior to the power outage",
-					"value": 2
-				}
-			]
+			"$import": "~/templates/master_template.json#state_after_power_failure_prev_on_off"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen14.json]: This should be the _off_on_prev variant, no?

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Looks good, just one detail

File: packages/config/config/devices/0x027a/zen14.json (hunk 1 above)
Comment (alvinchen1) [reply packages/config/config/devices/0x027a/zen14.json]: Confirmed.  Correct.

**Comment by alvinchen1** [comment]
Good to go with your changes.  Also, upon merge, can close https://github.com/zwave-js/node-zwave-js/pull/4921

Thanks!

**Review by alvinchen1 — COMMENTED** [review]
Looking good.

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks! This PR is merged, the original one will be merged in a minute or so.

**Pull request merged**